tests/sandbox_test.rs

```diff
@@ -52,8 +73,141 @@ mod tests {
             256,
         )
         .unwrap();
+        println!("Made market.");
+
+        let maker = Participant::new(
+            &sandbox,
+            &market_creator,
+            &market,
+            10 * LAMPORTS_PER_SOL,
+            1000,
+            2000,
+        )
+        .unwrap();
+        let taker = Participant::new(
+            &sandbox,
+            &market_creator,
+            &market,
+            10 * LAMPORTS_PER_SOL,
+            1000,
+            2000,
+        )
+        .unwrap();
+
+        // Place ask order
+        let taker_order = market
+            .new_order(
+                &taker.quote(),
+                &taker,
+                Side::Bid,
+                NonZeroU64::new(200).unwrap(),
+                OrderType::Limit,
+                NonZeroU64::new(500).unwrap(),
+                1,
+                SelfTradeBehavior::DecrementTake,
+                1,
+                NonZeroU64::new(300).unwrap(),
+                None,
+            )
+            .unwrap();
+
+        println!("Placed bid order.");
+
+        let maker_order = market
+            .new_order(
+                &maker.base(),
+                &maker,
+                Side::Ask,
+                NonZeroU64::new(200).unwrap(),
+                OrderType::Limit,
+                NonZeroU64::new(500).unwrap(),
+                1,
+                SelfTradeBehavior::DecrementTake,
+                1,
+                NonZeroU64::new(300).unwrap(),
+                None,
+            )
+            .unwrap();
+
+        println!("Placed ask order.");
+
+        let path = "./keypair";
+        solana_sdk::signature::write_keypair_file(&market_creator.keypair(), &path);
```

Comment:
You can use https://github.com/foonetic/solarium/blob/main/src/actor.rs#L51 which writes the keypair to the sandbox temp directory so you don't need to worry about cleaning it up

(didn't know about this function - can you change the Actor code to use it?)